doc/_scripts/redirects.py

```diff
@@ -281,7 +281,6 @@
     ('samples/boards/stm32/power_mgmt/adc/README', 'samples/boards/st/power_mgmt/adc/README'),
     ('samples/boards/stm32/power_mgmt/blinky/README', 'samples/boards/st/power_mgmt/blinky/README'),
     ('samples/boards/stm32/power_mgmt/serial_wakeup/README', 'samples/boards/st/power_mgmt/serial_wakeup/README'),
-    ('samples/boards/stm32/power_mgmt/standby_shutdown/README', 'samples/boards/st/power_mgmt/standby_shutdown/README'),
```

Comment:
Mmmh yeah no, you need a _new_ entry to redirect current page (which will be 404 for people coming from e..g old versions of the documentation) to.something else (likely `samples/boards/stm32/README`)